corehq/util/context_processors.py

```diff
@@ -66,7 +66,7 @@ def current_url_name(request):
 
 
 def js_api_keys(request):
-    if not request.analytics_enabled:
+    if hasattr(request, 'couch_user') and not request.couch_user.analytics_enabled:
```

Comment:
I'm curious why `request.analytics_enabled` was not available here? I thought we had added it in middleware, and that it returned essentially what you are doing here.